lib/parse.js

```diff
@@ -194,7 +194,7 @@ JS_Parse_Error.prototype.toString = function() {
 };
 
 function js_error(message, filename, line, col, pos) {
-    throw new JS_Parse_Error(message, line, col, pos);
+    throw new JS_Parse_Error(''+ message + ' at ' + filename, line, col, pos);
```

Comment:
Really? There are other places that use single quotes in that file. 

If that is really necessary maybe it should be fixed completely at other places, too. However maybe that's not the point of this issue?